README.md

```diff
@@ -101,6 +101,27 @@ In the GNU C Library the rounding mode can be controlled via the [`fesetround` a
 
 If strict compliance is sought with WebAssembly, then the user of Fizzy must ensure to keep the default rounding mode.
 
+
+## Development
+
+### Performance testing
+
+We use the following build configuration for performance testing:
+
+- Release build type (`-DCMAKE_BUILD_TYPE=Release` which enables `-O3` and disables asserts)
+
+- Latest version of GCC compiler (so far produces faster binaries than Clang)
```

Comment:
I'd change it to something like: "As of GCC 8, produces better results than Clang <= 10".